src/Models/Link.php

```diff
@@ -290,4 +296,23 @@ private function getLinkTypes(): array
 
         return $types;
     }
+
+    public function getDefaultTitle(): string
+    {
+        // If we have a title or don't have link, we can just bail out without any changes
+        if (!$this->config()->get('use_default_title') || empty($this->getURL()) || $this->Title) {
+            return '';
+        }
```

Comment:
```suggestion
    public function getDisplayTitle(): string
    {
        // If we have a title or don't have link, we can just bail out without any changes
        if ($this->Title) {
            return $this->Title;
        }
```

Calling this `getDisplayTitle()` and returning the title if there is one means we can just use `$DisplayTitle` in the template - we don't need any conditional logic there.

It also makes it easier for people to get "the title or the default if there's no title" in their own code or custom templates.

Reply:
Updated